torch/jit/frontend.py

```diff
@@ -337,12 +337,15 @@ def build_Name(ctx, expr):
 
     @staticmethod
     def build_NameConstant(ctx, expr):
-        text = "True" if expr.value else "False"
-        r = ctx.make_range(expr.lineno, expr.col_offset, expr.col_offset + len(text))
-        if expr.value:
+        r = ctx.make_range(expr.lineno, expr.col_offset, expr.col_offset + len(str(expr.value)))
+        if expr.value is True:
             return TrueLiteral(r)
-        else:
+        elif expr.value is False:
             return FalseLiteral(r)
+        elif expr.value is None:
+            return Var(Ident(r, "None"))
```

Comment:
There's not a NoneLiteral? It is weird and not Pythonic to treat None as a variable.

Reply:
I think so far we don't have a NoneLiteral yet. This PR is to correct the python3's behavior of treating None and match it with the python2 syntax (which treats None as a variable as well). When we begin to work on supporting None, we could think about making None as a Token and have a NoneLiteral here.

Reply:
Does this PR in its current state enable any programs to work that didn't work before? I think we should do put the change to this code, along with the introduction of a NoneLiteral token and support for it into a single PR. It is certainly not correct to treat "None" as a variable. This allows some weird programs to start working:

```
# works in python 2:
None = 4
return None + 1
```